src/vm/comcallablewrapper.h

```diff
@@ -465,28 +495,6 @@ class ComCallWrapperTemplate
     static ComCallWrapperTemplate *CreateTemplateForInterface(MethodTable *pItfMT);
 
 private:
-    
-    enum ComCallWrapperTemplateFlags
```

Comment:
This is internal implementation detail. Moving this to a global scope does not look like an improvement.

Reply:
I would agree with that sentiment and I can move this back into the class, but then using the defined flag operators becomes a bit more complicated. I can definitely hide the `enum` in the class, but since it isn't exposed on the type at all, it is just cluttering the name space..